### ISSUE TITLE
Handy-GAN and Handy-PCA

### ISSUE THREAD

**Issue by relh** [rolpotamias/handy#1]
Do you guys plans to release the Handy-GAN and/or Handy-PCA models? 

Basically the means by which to infer Handy parameters from an image.

**Comment by relh** [comment]
I guess I didn't really read closely, I think I'm more curious about this part:

`In order to boost high fidelity appearance
reconstruction, we collect a set of “in-the-wild” images and
predict Handy pose, shape and texture parameters using the
pre-trained regression network`

This pre-trained regression network that I think this is referring to is this one:

`Such method
includes a ResNet50 image encoder, pre-trained on Ima-
geNet and two branches that regress the pose and shape la-
tent codes of our parametric model. Unlike previous meth-
ods, in order to also reconstruct the texture, we add two
extra branches that regress the latent space w of the texture
model and the camera configuration (s, t). We train our net-
work using a set of loss functions that enable accurate hand
pose, shape, and appearance estimation`

So I'm just curious if you're planning to release this.
